packages/react-native-bridge/ios/GutenbergBridgeDelegate.swift

```diff
@@ -22,7 +22,7 @@ public enum Capabilities: String {
     case xposts
     case unsupportedBlockEditor
     case canEnableUnsupportedBlockEditor
-    case audioBlock
+	case isAudioBlockMediaUploadEnabled
```

Comment:
Looks like this line has a longer indentation than the rest.

Reply:
Ah, thanks I switched this around ordering-wise in VSCode instead of XCode so probably that's the reason.

Reply:
This [commit](https://github.com/WordPress/gutenberg/pull/31966/commits/11a79e7cd70695b06ddaf0dcf81a63defd2066b7) addressed the issue.